docs/src/whatsnew/3.0.rst

```diff
@@ -617,8 +617,8 @@ v3.0.4 (22 July 2021)
 .. _xxHash: https://github.com/Cyan4973/xxHash
 .. _PyKE: https://pypi.org/project/scitools-pyke/
 .. _@owena11: https://github.com/owena11
-.. _GitHub: https://github.com/SciTools/iris/issues/new/choose
 .. _readthedocs: https://readthedocs.org/
+.. _GitHub: https://github.com/SciTools/iris/discussions
```

Comment:
The sentence that references this link is specifically about bug reports and feature requests.  Not sure it makes sense to go via discussions???

For these ones, this pattern does work:

```
_GitHub: :issue:`new/choose`
```

It seems to go wrong where there is more than one word in the "handle" though.

Reply:
> this pattern does work

Huh, maybe it doesn't: the build seems to work but produces a broken link, at least locally.

Reply:
This works 🎉 

```
 And finally, get in touch with us on :issue:`GitHub<new/choose>` if you have any issues or  feature requests for improving Iris. Enjoy!
```